src/audio/module_adapter/module_adapter_ipc3.c

```diff
@@ -68,14 +68,29 @@ int module_adapter_init_data(struct comp_dev *dev,
 		data = spec;
 		break;
 	}
-	default:
+	case SOF_COMP_EQ_IIR:
+	case SOF_COMP_EQ_FIR:
+	case SOF_COMP_KEYWORD_DETECT:
+	case SOF_COMP_KPB:
+	case SOF_COMP_SELECTOR:
+	case SOF_COMP_DEMUX:
+	case SOF_COMP_MUX:
+	case SOF_COMP_DCBLOCK:
+	case SOF_COMP_SMART_AMP:
+	case SOF_COMP_MODULE_ADAPTER:
+	case SOF_COMP_NONE:
 	{
 		const struct ipc_config_process *ipc_module_adapter = spec;
 
 		size = ipc_module_adapter->size;
 		data = ipc_module_adapter->data;
 		break;
 	}
+	default:
+	{
```

Comment:
please remove braces

Reply:
@lyakh the whole switch case has them, do you want me to clean them all out?

Reply:
@cujomalainey other locations have them to be able to define local variables there. That is a not a pretty construct in general, but unfortunately C doesn't allow such definitions otherwise. One "work-around" once mentioned by @marc-hb was to use a stand-alone semicolon:
```
switch (x) {
case 0:
    ;
    int err;
    ...
}
```
I actually would rather have this even though it isn't much prettier. But at least (1) it shows clearly that we're working around something silly here, (2) avoids indentation confusion, (3) eliminates the impression of braces being a part of `case` clauses. Anyway, at least here under `default` you don't have any local variables, so those braces aren't needed. And since they aren't a part of `switch`-`case` syntax they can be removed just here - regardless of what is done with the rest. As for whether you change anything in the rest of this `switch`-`case` - it's up to you. I just thought that here in particular there was no good justification for them.

Reply:
> One "work-around" once mentioned by @marc-hb was to use a stand-alone semicolon:

Not just "mentioned" but merged in:
- https://github.com/zephyrproject-rtos/zephyr/pull/62232 (commit https://github.com/zephyrproject-rtos/zephyr/commit/f0fd9f1713426397d120897d5aadb2456fc8e9d3)

> Anyway, at least here under default you don't have any local variables, so those braces aren't needed.

+1

> @lyakh the whole switch case has them, do you want me to clean them all out?

My unsolicited opinion: I don't think you should touch these.